src/service/extension-script.js

```diff
@@ -155,7 +155,35 @@ export function createExtensionScript(win, extensionId, version) {
     version,
     getMode(win).localDev
   );
-  scriptElement.src = scriptSrc;
+
+  let policy = {
+    createScriptURL: function (url) {
+      // Only allow trusted URLs
+      const regexURL = new RegExp(
+        // eslint-disable-next-line local/no-forbidden-terms
+        '^https://([a-zA-Z0-9_-]+.)?cdn.ampproject.org(/.*)?$'
```

Comment:
could we move the `new RegExp` creation to the top level OR could we change them into expression literals since there doesnt seem to be anything dynamic about the pattern?

Reply:
I've moved it to the top level for now.